### ISSUE TITLE
[nautilus extension] TypeError: get_file_items must return a sequence of Nautilus.MenuItem

### ISSUE THREAD

**Issue by genodeftest** [GSConnect/gnome-shell-extension-gsconnect#1885]
### Describe the bug

This is an error reported to syslog, but I could not identify any user-visible problems.

### Steps to reproduce

1. Have the GNOME shell extension GSConnect installed
2. Open a nautilus window
3. Select a file
4. Right-click
5. Have a look at the syslog (for example by executing `journalctl --since=-120s`)

### Expected behavior

No errors or warnings

### GSConnect version

58

### Installed from

other source

### GNOME Shell version

gnome-shell-47.1-1.fc41.x86_64

### Linux distribution/release

Fedora 41

### Paired device(s)

(irrelevant)

### KDE Connect app version

1.32.9

### Support log

```plain text
Output from `journalctl --since=-60s`:

Exception ignored in PyObject_HasAttrString(); consider using PyObject_HasAttrStringWithError(), PyObject_GetOptionalAttrString() or PyObject_GetAttrString():
TypeError: get_file_items must return a sequence of Nautilus.MenuItem
```

### Screenshots

not applicable (no error visible)

### Notes

This error started happening with nautilus 47, it did not happen with nautilus 46 before. However, I don't see how the Nautilus API would have changed, as [upstream's git claims that the last commit was 9 months ago](https://gitlab.gnome.org/GNOME/nautilus/-/tree/main/libnautilus-extension).

I started thinking that my distribution might have modified the file, but this is not correct: `/usr/share/nautilus-python/extensions/nautilus-gsconnect.py` is identical with the version in the git repo.

**Comment by ferdnyc** [comment]
> I don't see how the Nautilus API would have changed

You'd also have to look for changes in [nautilus-python](https://gitlab.gnome.org/GNOME/nautilus-python), since the `TypeError` there is a Python `Exception` subclass. But I don't see anything relevant there in way longer than 9 months.

**Comment by genodeftest** [comment]
> Dumb question, but: Are you **sure** it's GSConnect, causing those messages? Could it be some other `nautilus-python` extension?

That's a very good question. I did not think I had any other `nautilus-python` extension installed, but I was wrong. There was another file in `~/.local/share/nautilus-python/extensions`, which I just deleted. And this made the problem go away.

It looks like the problem was caused by a python file from [Snoop](https://gitlab.gnome.org/philippun1/snoop), not nautilus (but GNOME's Gitlab doesn't currently work, so I'm unable to report the issue there at the time). I'm sorry for wasting your time.
